api/database/data/qa/01_non_log_data.sql

```diff
@@ -0,0 +1,160 @@
+('Bridge Farm', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers')),
+('The Bull', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers')),
+('Woodbine', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers')),
+('Lower Loxley Hall', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers')),
+('Simpsons Home', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Simpsons')),
+('Springfield Elementary', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Simpsons')),
+('Moes Tavern', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Simpsons')),
+('Krusty Burger', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Simpsons')),
+('Kwik-E-Mart', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Simpsons')),
+('Toad Hall', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Wind in the Willows')),
+('River Boat', (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Wind in the Willows'));
+
+
+----INSERT USER_ACCOUNT----
+INSERT INTO user_account
+  (user_name, gender_id, disability_id, ethnicity_id, email, phone_number, birth_year)
+VALUES
+('David Archer', 2,3,1,'avclung@gmail.com',+440000000001,1959),
+('Ruth Archer',1,3,1,'a@archers.com',+440000000002,1964),
+('Pip Archer',1,3,1,'b@archers.com',+440000000003,1991),
+('Johnny Phillips',2,3,1,'c@archers.com',+440000000004,1998),
+('Rex Fairbrother',2,3,1,'d@archers.com',+440000000005,1987),
+('Emma Grundy',1,3,1,'e@archers.com',+440000000006,1984),
+('Elizabeth Pargetter',1,3,1,'f@archers.com',+440000000007,1971),
+('Harrison Burns',3,3,1,'g@archers.com',+440000000008,1981),
+('Fallon Rogers',3,3,1,'h@archers.com',+440000000009,1977),
+('Lynda Snell',1,3,1,'i@archers.com',+440000000010,1959),
+('Marge Simpson',1,3,1,'avclung@yahoo.com.au',+440000000011,1982),
+('Lisa Simpson',1,3,1,'a@simpsons.com',+440000000012,2011),
+('Homer Simpson',2,3,1,'b@simpsons.com',+440000000013,1979),
+('Bart Simpson',2,3,1,'c@simpsons.com',+440000000014,2007),
+('Moe Szyslyak',2,3,1,'d@simpsons.com',+440000000015,1964),
+('Krusty the Clown',3,3,1,'e@simpsons.com',+440000000016,1969),
+('Apu Nahasapeemapetilon',3,3,1,'f@simpsons.com',+440000000017,1974),
+('Seymour Skinner',2,3,1,'g@simpsons.com',+440000000018,1973),
+('Selma Bouvier',3,3,1,'h@simpsons.com',+440000000019,1971),
+('Patty Bouvier',3,3,1,'i@simpsons.com',+440000000020,1971),
+('Ratty Rat',3,3,1,'admin@witw.com',+440000000021,1979),
+('Moly Mole',2,3,1,'a@witw.com',+440000000022,1978),
+('Mr Toad',2,3,1,'b@witw.com',+440000000023,1977),
+('Mr Badger',1,3,1,'c@witw.com',+440000000024,1976);
+
+
+----Adding User Access Roles----
```

Comment:
There are no `CB_ADMIN` accounts created for any of the cbs

Reply:
Volunteer admins can log into the dashboard.